hedera-node/src/main/java/com/hedera/services/ledger/BalanceChange.java

```diff
@@ -328,6 +343,7 @@ public String toString() {
                     .add("serialNo", aggregatedUnits)
                     .add("from", account)
                     .add("to", Id.fromGrpcAccount(counterPartyAccountId))
+                    .add("counterPartyAlias", counterPartyAlias.toStringUtf8())
```

Comment:
Imo `CommonUtils.hex(counterPartyAlias.getByteArray())` will be a more useful representation.